src/components/Logo/Logo.stories.mdx

```diff
@@ -102,7 +102,7 @@ export const logoRow = (logo, opts = {}) => {
   const { size = "large", displayValue } = opts;
   let key = logo;
   if (logo.indexOf("White") !== -1 || logo.indexOf("Negative") !== -1) {
-    styles.backgroundColor = "var(--nypl-colors-ui-gray-xdark)";
```

Comment:
There are other instances of `xdark` that need to be updated: `breadcrumbs.ts`, `button.ts`, and `hero.ts`.

Reply:
Good catch.  I had the scope too narrow on my multi-file search and I missed these.